packages/wallet/frontend/src/pages/transactions.tsx

```diff
@@ -0,0 +1,356 @@
+                }
+              }
+            }}
+          />
+        </div>
+        <div className="md:col-span-3 lg:col-span-2">
+          <Select
+            options={types}
+            label="Type"
+            placeholder="Select type..."
+            value={currentType}
+            onChange={(option) => {
+              if (option) {
+                redirect({ type: option.value, page: '0' })
+              }
+            }}
+          />
+        </div>
+        <div className="md:col-span-3 lg:col-span-2">
+          <Select
+            options={statuses}
+            label="Status"
+            placeholder="Select status..."
+            value={currentStatus}
+            onChange={(option) => {
+              if (option) {
+                redirect({ status: option.value, page: '0' })
+              }
+            }}
+          />
+        </div>
+      </div>
+
+      {error ? (
+        <div className="flex w-full flex-col items-center justify-center">
+          <p className="text-lg">{error}</p>
+          <Button
+            aria-label="refresh transactions table"
+            intent="secondary"
+            onClick={() => fetch(filters, pagination)}
+          >
+            Refresh table
+          </Button>
+        </div>
+      ) : loading ? (
+        <Table.Shimmer />
+      ) : (
+        <div className="w-full">
+          <Table>
+            <Table.Head
+              columns={[
+                '',
+                'Account',
+                'Payment pointer',
+                'Description',
+                'Amount',
+                'Status',
+                'Date'
+              ]}
+            />
+            <Table.Body>
+              {transactions.results.length ? (
+                transactions.results.map((trx) => (
+                  <Table.Row key={trx.id}>
+                    <Table.Cell className="w-10">
+                      <Arrow
+                        direction={trx.type === 'INCOMING' ? 'down' : 'up'}
+                      />
+                    </Table.Cell>
+                    <Table.Cell>{trx.accountName}</Table.Cell>
+                    <Table.Cell className="whitespace-nowrap">
+                      {trx.paymentPointerUrl ?? ''}
+                    </Table.Cell>
+                    <Table.Cell>
+                      {trx.description ?? 'No description'}
+                    </Table.Cell>
+                    <Table.Cell className="whitespace-nowrap">
+                      {trx.createdAt}
+                    </Table.Cell>
+                    <Table.Cell>
+                      <Badge
+                        intent={getStatusBadgeIntent(trx.status)}
+                        size="md"
+                        text={trx.status}
+                      />
+                    </Table.Cell>
+                    <Table.Cell>
+                      {
+                        formatAmount({
+                          value: trx.value ?? 0,
+
```

Comment:
no need for extra empty rows

Reply:
Can you elaborate?

Reply:
I see empty rows after every object key/value